scripts/scenarios/cloud8/qa/no-ssl/qa-scenario-2a.yaml

```diff
@@ -42,57 +49,30 @@ proposals:
           "@@data2@@":
             devices:
             - "@@sbd_device_data@@"
+          "@@data3@@":
+            devices:
+            - "@@sbd_device_data@@"
       per_node:
         nodes:
           "@@data1@@":
             params: ''
           "@@data2@@":
             params: ''
+          "@@data3@@":
+            params: ''
   deployment:
     elements:
       pacemaker-cluster-member:
       - "@@data1@@"
       - "@@data2@@"
+      - "@@data3@@"
       hawk-server:
       - "@@data1@@"
       - "@@data2@@"
-
-- barclamp: pacemaker
-  name: network
```

Comment:
why did we loose a dedicated network cluster?

Reply:
we have only enough nodes for 2 clusters(with 3 nodes) I can remove data and add network back if you think its better coverage /scenario

Reply:
No. Lets leave it like this if there are no more nodes available.